ItemLibrary/Categories.cs

```diff
@@ -4,13 +4,9 @@ public class Categories
     {
         public enum ItemCategory
         {
-            Computer
-        };
-
-        public enum ComputerCategory
-        {
-            Desktop,
-            Laptop
+            Computer,
+            Smartphone,
+            Desktop
```

Comment:
I think that desktop can be misleading. Maybe better DesktopComputer or something else

Reply:
changed naming